libsolidity/formal/SMTEncoder.cpp

```diff
@@ -1358,7 +1358,7 @@ bool SMTEncoder::visit(MemberAccess const& _memberAccess)
 	{
 		auto const* functionType = dynamic_cast<FunctionType const*>(_memberAccess.annotation().type);
 		if (functionType && functionType->hasDeclaration())
-			defineExpr(_memberAccess, functionType->externalIdentifier());
+			defineExpr(_memberAccess, u256(_memberAccess.id()));
```

Comment:
Looks like it works and fixes the bug but I have some doubts about this:
1. Is it correct to use AST node id in this case?
2. cast from int_64 to u256.